db/migrate/20230406182937_create_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMemberships < ActiveRecord::Migration[7.0]
+  def change
+    create_table :memberships do |t|
+      t.belongs_to :user, null: false, foreign_key: true
+      t.belongs_to :clubhouse, null: false, foreign_key: true
+      t.string :role, null: false, default: Membership::MEMBER_ROLE
```

Comment:
There is also this alternative of modeling the role as an enum column. https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html

I'm not a super fan of it, but there it is, if you want to use it

Reply:
I like the way we have it